packages/babel-helpers/src/helpers.js

```diff
@@ -1717,3 +1717,12 @@ helpers.decorate = helper("7.0.1")`
     return constructor;
   }
   `;
+
+helpers.classPrivateMethodGet = helper("7.0.0-beta.0")`
+  export default function _classPrivateMethodGet(receiver, privateMap) {
```

Comment:
Can't we reuse the `classPrivateFieldGet` helper? Since with decorators methods could become fields and vice versa, I'd prefer fields/methods/accessors to be handled the same way.

Reply:
This was recently changed when I switched to using `WeakSet`s.